projects/aca-content/src/lib/ui/dynamic-theme/typography.scss

```diff
@@ -2,78 +2,93 @@
+      $body-1: mat.define-typography-level(1rem, 1.42rem, 400),
+      $caption: mat.define-typography-level(0.86rem, 1.42rem, 400),
+      $button: mat.define-typography-level(1rem, 1rem, 500),
+      $font-family: $default-font-family,
+      $input: mat.define-typography-level(1.14em, 1.25, 400)
     );
+  }
 
-    @if $base-font-size {
-        $custom-typography: mat.define-typography-config(
-            $display-4:     mat.define-typography-level(8rem, 8rem, 300),
-            $display-3:     mat.define-typography-level(4rem, 4rem, 400),
-            $display-2:     mat.define-typography-level(3.21rem, 3.21rem, 400),
-            $display-1:     mat.define-typography-level(2.42rem, 2.85rem, 400),
-            $headline:      mat.define-typography-level(1.71rem, 2.28rem, 400),
-            $title:         mat.define-typography-level(1.42rem, 2.28rem, 500),
-            $subheading-2:  mat.define-typography-level(1.14rem, 2rem, 400),
-            $subheading-1:  mat.define-typography-level(1.07rem, 1.71rem, 400),
-            $body-2:        mat.define-typography-level(1rem, 1.71rem, 500),
-            $body-1:        mat.define-typography-level(1rem, 1.42rem, 400),
-            $caption:       mat.define-typography-level(0.86rem, 1.42rem, 400),
-            $button:        mat.define-typography-level(1rem, 1rem, 500),
-            $font-family:   $default-font-family,
-            $input:         mat.define-typography-level(1.14em, 1.25, 400)
+  @if $font-family {
+    @each $key, $level in $custom-typography {
+      @if type-of($level) == 'map' {
+        $new-level: map-merge(
+          $level,
+          (
+            font-family: $font-family
+          )
+        );
+        $custom-typography: map-merge(
+          $custom-typography,
+          (
+            $key: $new-level
+          )
         );
+      }
     }
 
-    @if $font-family {
-        @each $key, $level in $custom-typography {
-            @if type-of($level) == 'map' {
-                $new-level: map-merge($level, (font-family: $font-family));
-                $custom-typography: map-merge($custom-typography, ($key: $new-level));
-            }
-        }
-
-        $custom-typography: map-merge($custom-typography, (font-family: $font-family));
-    }
+    $custom-typography: map-merge(
+      $custom-typography,
+      (
+        font-family: $font-family
+      )
+    );
+  }
 
-    @return $custom-typography;
-};
+  @return $custom-typography;
+}
 
 @function get-custom-adf-font-sizes() {
-    @return (
-        'theme-adf-icon-1-font-size': 1.2rem,
-        'theme-adf-picture-1-font-size': 1.28rem,
-        'theme-adf-task-footer-font-size': 1.28rem,
-        'theme-adf-task-title-font-size': 1.28rem
-    )
-};
+  @return (
+    'theme-adf-icon-1-font-size': 1.2rem,
+    'theme-adf-picture-1-font-size': 1.28rem,
+    'theme-adf-task-footer-font-size': 1.28rem,
+    'theme-adf-task-title-font-size': 1.28rem,
+    'theme-adf-spacing': 16px
```

Comment:
```'theme-adf-spacing': 16px```

This is the only change provided by me, all other changes are indentation etc. done while performing commit